docs/self-managed/setup/upgrade.md

```diff
@@ -2,130 +2,61 @@
+## Upgrade requirements
 
-```shell
-helm upgrade camunda
-```
-
-However, if Camunda Identity is enabled (which is the default), the upgrade path is a bit more complex than just running `helm upgrade`. Read the next section to familiarize yourself with the upgrade process.
-
-### Upgrading where Identity enabled
-
-If you have installed the Camunda 8 Helm charts before with default values, this means Identity and the related authentication mechanism are enabled. For authentication, the Helm charts generate the secrets randomly if not specified on installation for each web application. If you run `helm upgrade` to upgrade to a newer chart version, you likely will see the following return:
-
-```shell
-helm upgrade camunda-platform-test camunda/camunda-platform
-```
-
-You likely will see the following error:
-
-```shell
-Error: UPGRADE FAILED: execution error at (camunda-platform/charts/identity/templates/tasklist-secret.yaml:10:22):
-PASSWORDS ERROR: You must provide your current passwords when upgrading the release.
-                 Note that even after reinstallation, old credentials may be needed as they may be kept in persistent volume claims.
-                 Further information can be obtained at https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases
-
-    'global.identity.auth.tasklist.existingSecret' must not be empty, please add '--set global.identity.auth.tasklist.existingSecret=$TASKLIST_SECRET' to the command. To get the current value:
-
-        export TASKLIST_SECRET=$(kubectl get secret --namespace "camunda" "camunda-platform-test-tasklist-identity-secret" -o jsonpath="{.data.tasklist-secret}" | base64 --decode)
-```
-
-As mentioned, this output returns because secrets are randomly generated with the first Helm installation by default if not further specified. We use a library chart [provided by Bitnami](https://github.com/bitnami/charts/tree/master/bitnami/common) for this. The generated secrets persist on persistent volume claims (PVCs), which are not maintained by Helm.
-
-If you remove the Helm chart release or do an upgrade, PVCs are not removed nor recreated. On an upgrade, secrets can be recreated by Helm and could lead to the regeneration of the secret values. This would mean that newly generated secrets would no longer match with the persistent secrets. To avoid such an issue, Bitnami blocks the upgrade path and prints the help message as shown above.
-
-In the error message, Bitnami links to their [troubleshooting guide](https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases). However, to avoid confusion, we will step through the troubleshooting process in this guide as well.
-
-### Secrets extraction
+For a smooth upgrade experience, we recommend determining both your **Helm chart** and **Helm CLI** versions prior to starting your upgrade.
```

Comment:
Is this a very strong **recommendation**, or can we just say "For a smooth upgrade experience, determine your **Helm chart** and **Helm CLI** versions prior to starting your upgrade."? I try to stray away from "we recommend" if it is a stronger command, if that makes sense? See [this PR](https://github.com/camunda/camunda-docs/pull/3437)for some context.

Reply:
Maybe another distribution team member might have other opinions, but I don't think we're that strict on helm CLI versions.   I know from reading  commits on helm that they haven't added any new features in the past year, so if someones on ubuntu and installed `apt install helm` and that version of helm isn't like,  3 years old, I wouldn't expect issues. But the notice could be there simply to let people know the helm versions we're testing with.